superbench/benchmarks/model_benchmarks/model_base.py

```diff
@@ -265,8 +265,8 @@ def __train(self, precision):
         # The unit of step time should be millisecond.
         step_times = self._train_step(precision)
         if isinstance(step_times, tuple):
-            step_times = step_times[0]
             info = step_times[1]
+            step_times = step_times[0]
```

Comment:
Why change the order???

Reply:
there's a bug for previous order, the info will be the second step time
```
step_times = step_times[0]
info = step_times[1]
```

Reply:
got it